airbyte-cdk/python/airbyte_cdk/sources/declarative/low_code_component_schema.yaml

```diff
@@ -1,918 +0,0 @@
-"$schema": http://json-schema.org/draft-07/schema#
```

Comment:
we had renamed the schema to `declarative_component_schema.yaml` but this file was not removed. Just getting rid of it since it's unnecessary.